src/MonologCreator/Factory.php

```diff
@@ -6,55 +6,49 @@
 use Monolog;
 
 /**
- * Factory class to for creating monolog loggers with preconfigurated array
+ * Factory class to for creating monolog loggers with pre-configured array
  */
 class Factory
 {
     /**
      * @var array
      */
-    private $config = array();
+    private $levels = [
```

Comment:
why do we re-map them, if they exist already in Monolog?

Reply:
good question. Currently it is used as mapping between the value you can set in the config array and the internal value Monolog uses. I think it might be obsolete now, as the following function could be used instead: https://github.com/Seldaek/monolog/blob/main/src/Monolog/Level.php#L88

I will test this.

Reply:
I've removed the level mapping now, as it can be handled be the `Level` enum now.